src/vs/workbench/parts/files/electron-browser/fileActions.contribution.ts

```diff
@@ -396,7 +396,7 @@ MenuRegistry.appendMenuItem(MenuId.ExplorerContext, {
 		id: OPEN_FOLDER_SETTINGS_COMMAND,
 		title: OPEN_FOLDER_SETTINGS_LABEL
 	},
```

Comment:
No. This will then also work for the top level singe folder case. Which we do not want.
Same for the next change.